tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -209,6 +222,52 @@ def __init__(self, library: Library, driver: "QtDriver"):
             (self.image_container.size().width(), self.image_container.size().height())
         )
 
+        root_layout = QHBoxLayout(self)
+        root_layout.setContentsMargins(0, 0, 0, 0)
+        root_layout.addWidget(splitter)
+
+    def fill_libs_widget(self, layout: QVBoxLayout):
+        settings = self.driver.settings
+        settings.beginGroup(SettingItems.LIBS_LIST)
+        lib_items = {}
+        for key in settings.allKeys():
+            val = settings.value(key)
+            if len(val) > 45:
+                val = f"{val[0:10]} ... {val[-10:]}"
+            lib_items[key] = val
+        settings.endGroup()
+        new_keys = set(lib_items.keys())
+        if new_keys == self.render_libs:
+            # no need to re-render
+            return
+        self.render_libs = new_keys
+        return self._fill_libs_widget(lib_items, layout)
+
+    def _fill_libs_widget(self, libraries: dict[str, str], layout: QVBoxLayout):
+        # remove any potential previous items
+        for idx in reversed(range(layout.count())):
+            widget = layout.itemAt(idx).widget()
+            layout.removeWidget(widget)
+            # remove from GUI
+            widget.setParent(None)
+
+        label = QLabel("Select a library")
```

Comment:
Suggestion to rename this section to "Recent Libraries". I feel this is more descriptive of the section, and is more consistent with similar "File -> Open Recent Thing" functionality found in other programs, and likely this program in the future.
```suggestion
        label = QLabel("Recent Libraries")
        label.setStyleSheet("font-weight:bold;")
```